spec/shared/contexts/movie_context.rb

```diff
@@ -216,7 +216,7 @@ class CachingMovieSerializer
       belongs_to :owner, record_type: :user
       belongs_to :movie_type
 
-      cache_options enabled: true
+      cache_options store: ActiveSupport::Cache::MemoryStore.new
```

Comment:
@doits let's add some extra options here, ex: `expires_in: 5`, I think the tests will fail based on the
https://github.com/fast-jsonapi/fast_jsonapi/pull/52/files#diff-0d22ea72f1e72c75e7887cdbcba46871R70

I think there's a missing __*__ in `record_hash = cache_store_instance.fetch(record, *cache_store_options)`

Reply:
Yeah, tests for all the new functionality are missing yet (and might reveal some more errors). I'll add tests if the implementation is ready.

Reply:
Sorry, let me know when I could take a final look :see_no_evil: 

Thanks again for bearing with me :heart: